### ISSUE TITLE
Document scope of support for dch-photon

### ISSUE THREAD

**Issue by stuclem** [vmware/vic-product#2165]
Per https://github.com/vmware/vic-planning/issues/115. We need to clearly list which use cases are supported and which are not.

**Comment by stuclem** [comment]
@vidya-v it looks like this is still in 1.5.3, but should be confirmed.

**Comment by stuclem** [comment]
Not in 1.5.4 so moving to Backlog.

**Comment by stuclem** [comment]
@renmaosheng are we going to address this in 1.5.5?

**Comment by renmaosheng** [comment]
No, we will not address this in 1.5.5, let's keep the current status. thanks.
